examples/examples_test.go

```diff
@@ -128,7 +128,7 @@ func validateObject(obj runtime.Object) (errors field.ErrorList) {
 		}
 		errors = expvalidation.ValidateDaemonSet(t)
 	default:
-		return field.ErrorList{field.InternalError(field.NewPath(""), fmt.Errorf("no validation defined for %#v", obj))}
+		return field.InternalError(field.NewPath(""), fmt.Errorf("no validation defined for %#v", obj))
```

Comment:
Oops, too many revisions going on.  I do see the vet error on my client:

$ head -131 examples/examples_test.go | tail -1
                return field.ErrorList{field.InternalError(field.NewPath(""), fmt.Errorf("no validation defined for %#v", obj))}

$ godep go vet ./examples/
godep: WARNING: Go version (go1.6) & $GO15VENDOREXPERIMENT= wants to enable the vendor experiment, but disabling because a Godep workspace (Godeps/_workspace) exists
examples/examples_test.go:131: field.ErrorList composite literal uses unkeyed fields
exit status 1
godep: go exit status 1